NEWS.md

```diff
@@ -55,3 +55,6 @@ Revisions
 
 * transfer maintainership
 
+# GeoDiff 1.1.3
+
+* implement code optimizations (mostly RFast and matrix methods) to speed up execution. Added error handling to Rcpp code.
```

Comment:
Can you add more explicit changes such as what files were changed? Alternatively, you could also just indicate this affects the package throughout.

Reply:
done